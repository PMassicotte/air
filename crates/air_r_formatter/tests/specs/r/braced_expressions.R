# Hard line break between `{}` tokens
function() {}
for (i in x) {}
if (a) {}

{}

{
  1
}

{
  # comment
}
