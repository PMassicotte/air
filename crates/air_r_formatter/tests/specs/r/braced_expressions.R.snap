---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/braced_expressions.R
---
# Input

```R
# Hard line break between `{}` tokens
function() {}
for (i in x) {}
if (a) {}

{}

{
  1
}

{
  # comment
}

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
-----

```R
# Hard line break between `{}` tokens
function() {
}
for (i in x) {
}
if (a) {
}

{
}

{
	1
}

{
	# comment
}
```
